src/ducks/apps/components/QuerystringSections.jsx

```diff
@@ -39,7 +39,13 @@ const parseURLSearchParams = urlParams => {
 
 const getFilterFromQuery = query => {
   const usp = new URLSearchParams(query)
-  return omit(parseURLSearchParams(usp), FILTER_BLACK_LIST)
+  const params = parseURLSearchParams(usp)
+  if (params.underMaintenance == undefined) {
+    params['underMaintenance'] = false
+  } else {
+    delete params.underMaintenance
+  }
+  return omit(params, FILTER_BLACK_LIST)
```

Comment:
Also we should use either `params['underMaintenance']` form or `params.underMaintenance` form, but not both (this makes the code harder to read)

Reply:
This makes it possible to hide applications by default without putting a filter in the url parameters. I don't know how to make it more explicit because the filters are managed by `cozy-ui` and I don't want to set an url parameter at the start. The other solution is refacto `cozy-ui` to repatriate app filtering in` cozy-store`.